deepmoji/create_vocab.py

```diff
@@ -113,15 +118,17 @@ def populate_master_vocab(self, vocab_path, min_words=1, force_appearance=None):
 
             sizes[path] = sum(dicts[path].values())
             print('Overall word count for {} -> {}'.format(path, sizes[path]))
-            print('Overall word number for {} -> {}'.format(path, len(dicts[path])))
+            print(
+                'Overall word number for {} -> {}'.format(path, len(dicts[path])))
```

Comment:
The line wrapping in PEP8 is a bit excessive.. This does not seem more readable. Could you disable the line wrapping in flake8 and rerun it?